h2/src/main/org/h2/mvstore/FileStore.java

```diff
@@ -186,21 +186,20 @@ public void open(String fileName, boolean readOnly, char[] encryptionKey) {
      * Close this store.
      */
     public void close() {
-        if(file != null && file.isOpen()) {
-            try {
-                if (fileLock != null && fileLock.isValid()) {
+        try {
+            if(file != null && file.isOpen()) {
```

Comment:
I think that `file != null` check should be outside of the `try` statement, but difference is not significant.